Makefile

```diff
@@ -11,14 +11,15 @@ clean :
 	rm -rf $(BIN_DIR)
 
 run-dev : build
-	$(BIN_DIR)/habitat --procdir $(DEV_PROC_DIR)
+	$(BIN_DIR)/habitat --procdir $(DEV_PROC_DIR) --communitydir $(DEV_COMMUNITY_DIR) --hostname localhost
```

Comment:
eventually we want a central place where all the data is stored, and i think it should be separated by community (like data/community_1, data/community_2 etc.) so can we rename this to communities / support that now?

Reply:
I can rename it to communities. Right now, they are split up by community ID, but the directory structure is `data/communities/community_id`